controllers/nginx/configuration.md

```diff
@@ -162,37 +160,37 @@ By default NGINX uses `http` to reach the services. Adding the annotation `ingre
 
 ### Whitelist source range
 
-You can specify the allowed client ip source ranges through the `ingress.kubernetes.io/whitelist-source-range` annotation, eg;  `10.0.0.0/24,172.10.0.1`
-For a global restriction (any URL) is possible to use `whitelist-source-range` in the NGINX config map
+You can specify the allowed client ip source ranges through the `ingress.kubernetes.io/whitelist-source-range` annotation, eg  `10.0.0.0/24,172.10.0.1`.
+For a global restriction (any URL) is possible to use `whitelist-source-range` in the NGINX config map.
```

Comment:
reword.

suggest: "To configure this setting globally for all ingresses, the `whitelist-source-range` value may be set in the NGINX ConfigMap".